### PR TITLE
[RHDM-1668] Fix Serialization inside PMML

### PR DESCRIPTION
**Pull request by gitgabrio**
@danielezonca @kostola  @jiripetrlik

See https://issues.redhat.com/browse/RHDM-1668

### CODE REVIEW

**Comment by gitgabrio** [comment]
Jenkins run fdb

File: kie-pmml-trusty/kie-pmml-models/kie-pmml-models-regression/kie-pmml-models-regression-tests/src/test/java/org/kie/pmml/regression/tests/LinearRegressionSampleWithTransformationsTest.java [hunk 1]
```diff
@@ -72,10 +72,10 @@ public static void setupClass() {
     public static Collection<Object[]> data() {
         return Arrays.asList(new Object[][]{
                 {27, 34000, "street", 989.1},
-                {49, 78000, "carpark", 1301.37},
-                {57, 72000, "street", 1582.1},
-                {61, 123000, "carpark", 1836.5699999999997},
-                {18, 26000, "street", 845.1999999999999},
+//                {49, 78000, "carpark", 1301.37},
```
Comment (jstastny-cz) [inline kie-pmml-trusty/kie-pmml-models/kie-pmml-models-regression/kie-pmml-models-regression-tests/src/test/java/org/kie/pmml/regression/tests/LinearRegressionSampleWithTransformationsTest.java]: Is there a reason to keep these commented lines, and is the line above valid? given the trailing comma.

Comment (gitgabrio) [reply kie-pmml-trusty/kie-pmml-models/kie-pmml-models-regression/kie-pmml-models-regression-tests/src/test/java/org/kie/pmml/regression/tests/LinearRegressionSampleWithTransformationsTest.java]: @jstastny-cz - it is a left-over - just fixed. Btw, yes, that trailing comma is "valid" (even if bad) - I removed it

**Comment by gitgabrio** [comment]
Jenkins run cdb

**Review by jiripetrlik — APPROVED** [review]
LGTM

**Comment by gitgabrio** [comment]
@jstastny-cz @danielezonca 
Could you please look at this and, eventually, approve/merge ? I think it also has to be backported to 7.52.x

**Review by danielezonca — APPROVED** [review]

**Pull request merged**